mindee/parsing/common/extras/extras.py

```diff
@@ -0,0 +1,26 @@
+from abc import ABC
+
+from mindee.parsing.common.string_dict import StringDict
+
+
+class ExtraField(ABC):
+    """Extra utilities."""
+
+
+class Extras:
+    """
+    ExtraField collection wrapper class.
+
+    Is roughly equivalent to a dict of Extras, with a bit more utility.
+    """
+
+    def __init__(self, raw_prediction: StringDict):
+        for key, extra in raw_prediction.items():
+            setattr(self, key, extra)
+
+    def __str__(self) -> str:
+        out_str = ""
+        for attr in dir(self):
+            if not attr.startswith("__"):
+                out_str += f":{attr}:" + getattr(self, attr).__str__() + "\n"
```

Comment:
Yeah idk what linting tool did this, but your version was what I had initially. I think we'll update the pre-commit hooks in a later PR because it keeps doing stuff like this...